frescobaldi_app/vcs/gitdoc.py

```diff
@@ -298,6 +298,7 @@ def set_status(gitprocess, exitcode):
 
         # arguments to get file status
         args = [
+            # TODO: really? including ignored files?
```

Comment:
> Do you mean you collect this list in order to not check files that are in it? That makes sense, of course

`_update_status()` only checks the `Document()` object's corresponding file. This object will be added into the `Repo()`. But we don't track its git diff.  We can display a status line in the bottom E.g. `In Frescobaldi on rewrite-vcs, File is ignored.` That's my current thought.